app/src/common/shared/org/mozilla/vrbrowser/ui/KeyboardWidget.java

```diff
@@ -229,50 +261,59 @@ public void onPress(int primaryCode) {
     @Override
     public void onLongPress(Keyboard.Key popupKey) {
         if (popupKey.popupCharacters != null) {
-            mOriginalLayoutParams = (RelativeLayout.LayoutParams) mKeyboardview.getLayoutParams();
             RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(
                     RelativeLayout.LayoutParams.WRAP_CONTENT,
                     RelativeLayout.LayoutParams.WRAP_CONTENT);
 
+            boolean rightAligned = false;
+            int maxCharsPerLine = MAX_CHARS_PER_LINE_LONG;
             StringBuilder popupCharacters = new StringBuilder(popupKey.popupCharacters);
             switch (popupKey.codes[0]) {
+                case 110: {
```

Comment:
part of the refactor should be replacing these numbers with const values so it is easier to read.